pulseapi/entries/models.py

```diff
@@ -78,6 +78,7 @@ class Entry(models.Model):
     interest = models.CharField(max_length=600, blank=True)
     featured = models.BooleanField(default=False)
     internal_notes = models.TextField(blank=True)
+    posted_by_creator = models.BooleanField(default=False)
```

Comment:
nit: maybe use the same terminology we use for the person posting - `published_by`, so it would be `published_by_creator`?

Reply:
fair - let me update the code